docs/03-github/04-builder.mdx

```diff
@@ -466,11 +466,17 @@ _**required:** `false`_ _**default:** `""`_
 Amount of memory to assign the docker container. Defaults to 95% of total system memory rounded down
 to the nearest megabyte on Linux and 80% on Windows when no value is specified. On unrecognized
 platforms, defaults to 75% of total system memory. To manually specify a value, use the format
-
 \<number\>\<unit\>, where unit is either m or g. ie: 512m = 512 megabytes and 4g = 4 gigabytes.
 
 _**required:** `false`_ _**default:** `""`_
 
+#### dockerIsolationMode
+
+Isolation mode to use for the docker container. Can be one of process, hyperv, or default. Only
+applicable on Windows.
+
+_**required:** `false`_ _**default:** `"default"`_
+
```

Comment:
Do we want to pick a sensible default instead of making this a parameter?

Reply:
So these are actually docker's own parameters. The reason I'm going with default is because on windows server it defaults to process isolation and on desktop it defaults to hyperv. Since I don't want to inadvertently break people's self hosted runners by forcing process mode I think leaving it as default is best. Unsure the side effects of forcing process mode so not sure about making that leap.